### ISSUE TITLE
Element remains selected, even after items in dropdown change

### ISSUE THREAD

**Issue by Parasin** [Juravenator/paper-dropdown-input#4]
I am attempting to use the component to initiate state transitions on an object in my application.
The first time around, it works as expected; I click the list, it opens, and the items are displayed appropriately:
![1](https://cloud.githubusercontent.com/assets/10092035/25437619/93f54472-2a5c-11e7-9f35-2687efc052fa.png)

I select the item that matches what that state transition to be performed, and then click the list again:
![2](https://cloud.githubusercontent.com/assets/10092035/25437647/aa95b3ba-2a5c-11e7-8d6f-a751168a018b.png)
You notice that "**Resume**" is already highlighted, incorrectly? How can I either disable this, or remove this behavior?

I then click "**Resume**", but nothing happens, even though the `selected-item` should now be "Resume":
![3](https://cloud.githubusercontent.com/assets/10092035/25437993/c0588172-2a5d-11e7-854e-d2f092b098f1.png)

Here is the code that I am using for the dropdown:
```
<paper-dropdown-input 
          id="stateDropdown"
          label="State"
          items="{{ availableStates }}"
          filter-property="value"
          placeholder="{{ task.status.value }}"
          selected-item="{{ selectedState }}">
</paper-dropdown-input>
```

**Comment by Juravenator** [comment]
Hi @Parasin, thanks for the report

The `selectedItem` indeed wasn't properly unset when the `items` array changes. This should be fixed with 18f882f58c8510d88d0d6a39b413890aa175a2bf.

Could you confirm this fix? If this fixes it, I will tag a new release.

**Comment by Parasin** [comment]
Hey @Juravenator really appreciate the hot-fix, it works like a charm!
I have tested it out and it appears to work as expected. The only change that I had to make in my HTML is shown below:

```
<paper-dropdown-input id="stateDropdown"
                                         label="State"
                                         items="{{ availableStates }}"
                                         filter-property="value"
                                         placeholder="{{ task.status.value }}"
                                         selected-item="{{ selectedState }}">
     <template>
              <template is="dom-repeat" items="[[items]]" as="item">
                                        <paper-item>[[ item.value ]]</paper-item>
              </template>
     </template>
</paper-dropdown-input>
```

Thank you again :+1:  
I really appreciate the effort and this great component!
